Cargo.toml

```diff
@@ -10,3 +10,5 @@ bitflags = "1.3.2"
 lazy_static = "1.4.0"
 local-macros = { path = "local-macros" }
 regex = "1.5.4"
+serde = "1.0.133"
```

Comment:
I think you can specify only needed parts of `serde` to make compilation faster?